src/test/java/core/WebDriverSingleton.java

```diff
@@ -0,0 +1,19 @@
+package core;
+
+import org.openqa.selenium.WebDriver;
+
+public class WebDriverSingleton {
+    private static WebDriver driver;
```

Comment:
what if in the future we will run our tests in several threads? please think on how to make this driver instance thread safe (hint - ThreadLocal)